container-runtime/crun/pkg.yaml

```diff
@@ -0,0 +1,40 @@
+name: crun
+variant: scratch
+shell: /toolchain/bin/bash
+dependencies:
+  - stage: base
+steps:
+  - sources:
+      # {{ if eq .ARCH "aarch64" }} This in fact is YAML comment, but Go templating instruction is evaluated by bldr
+      - url: https://github.com/containers/crun/releases/download/{{ .CRUN_VERSION }}/crun-{{ .CRUN_VERSION }}-linux-arm64-disable-systemd
```

Comment:
we should also build from source probably

Reply:
When is a requirement I can change to building from source. I'm already building crun from source with WASM enabled for another project so this should not be a problem. Just wanted to keep things simple.

Just one question: Do I need to handle any amd64/arm64 specific steps to build from source for both platforms or is the container build for both anyway an I can just assume I run on the right ARCH?

Version schema should follow talos upstream version. Is this correct?

Reply:
I tried building it with the talos build environment. But I got some errors:

 * The crun release source tar.gz package does not contain the submodule code - so I need to clone the repo tag with submodules, annoying but not a problem.
 * Autogen does not find `libtoolize` - is there a way to install additional packages? Build ENV does seem to be based debian/alpine based.

Just for reference, on debian I install the following tools to install it:
```bash
apt-get install curl gcc build-essential pkgconf m4 libtool libsystemd-dev \
    libprotobuf-c-dev libcap-dev libseccomp-dev libyajl-dev libyajl2 \
    libprotobuf-c-dev protobuf-c-compiler protobuf-compiler libgcrypt20-dev \
    python3-protobuf go-md2man autoconf python3 automake \
    golang-github-checkpoint-restore-go-criu-dev
```

Reply:
this sounds like a lot of dependencies, probably let's skip building from source for now

Reply:
i think we have all the dependencies, i;ll take a look before re-signing, otherwise will fall back to binaries from release

Reply:
we're missing just yajl in our build environment, not sure if we add as a dependency and build or just use the static binaries